✨ UI/UX Improvements – Homepage Redesign

mbagalwa opened the pull request:
Hello @Baraka24 @Shadow243 👋🏻  
Hope you're doing great! 🚀 Here’s the updated homepage redesign with improved structure, styles, and a smoother UI/UX.

✨ UI/UX Improvements – Homepage Redesign

## 📌 Description
Homepage redesign with a clearer structure, new styles, and better UI/UX consistency.  
Added Hero, Features, Testimonials, and FAQ sections.

## 🔧 Main Changes
- **Hero section**: new title, subtitle, CTAs, mini-stats + avatar group.  
- **About**: added a simple and clear descriptive block.  
- **Features**: responsive grid with harmonized cards.  
- **Testimonials**: standardized cards with theme and animations.  
- **FAQ**: ready-to-use structure with accordion styles.  

## 📂 Files Changed
- `pages/index.md`  
- `static/assets/home/index.css`  
- `static/assets/home/features.css`  
- `static/assets/home/testimonials.css`  
- `static/assets/home/label.css`  
- `static/assets/main.css`  

## Overview
   

- [ ]  Before
       
https://github.com/user-attachments/assets/f1564559-82ff-4b75-91b6-3e1edb8e6b33

- [ ]  After
    
https://github.com/user-attachments/assets/3e18e04b-c5f3-4919-b025-2e7820becf25

mbagalwa committed:
feat: modern UI redesign - glassmorphism navbar and responsive footer

mbagalwa committed:
feat: modern UI redesign - glassmorphism navbar and responsive footer

mbagalwa committed:
build hero & about section

mbagalwa committed:
build home page with 5 sections(heropage, about, features, testo,omies, faq)

Shadow243 commented:
@mbagalwa this https://github.com/cypht-org/cypht-website/pull/134 should be then closed ?

mbagalwa commented:
Hello @marclaporte , I didn't use a template, I started from scratch based on UI/UX examples on Pinterest, dribbling, drawing on paper and finally realizing it with CSS

marclaporte commented:
> I didn't use a template

It looks great!!!

mbagalwa commented:
Glad to hear it 😇

Baraka24 committed:
Merge branch 'master' into feat/home-page-redisign

mbagalwa committed:
build features page (first proposal)

mbagalwa committed:
review features ui/ux

mbagalwa committed:
build feature page

mbagalwa committed:
build securty page structure

mbagalwa committed:
review feature, security  and module heropages

marclaporte commented on layouts/partials/navbar.html.twig:
```diff
@@ -0,0 +1,213 @@
+            title: 'Features', 
+            href: '/features', 
+            icon: 'bi-star',
+            description: 'Fonctionnalités principales'
```
Please prepare everything in English.

mbagalwa replied:
Alright, I’ll make sure every word and sentence is in English

mbagalwa committed:
redesign ui/ux for security page

mbagalwa committed:
build module page

mbagalwa committed:
build module page

Baraka24 committed:
Merge branch 'master' into feat/home-page-redisign

mbagalwa committed:
fix conflit, change chip-module-card style

mbagalwa commented:
Hey 👋🏻

I have modified the Features, Security, and Modules pages.
Note: I still need to find suitable images or illustrations for the different cards on the Features page.

1. Features 
   +- Before
   https://github.com/user-attachments/assets/6e2ce25a-3f84-4f76-a4b5-67e58d7a8e12

   +- After
    https://github.com/user-attachments/assets/1d9d2ef3-024c-40c8-a601-008359679915

2. Security
   +- Before
   https://github.com/user-attachments/assets/5e8cca8c-0998-4124-93e9-9535f8a73c0f

   +- After
    https://github.com/user-attachments/assets/0e968827-34c7-4e07-a295-cf70625567bc

3. Module
   +- Before 
   https://github.com/user-attachments/assets/f2de60db-55a9-4cc1-b340-0f000188bff7

    +- After
   https://github.com/user-attachments/assets/783d7ad1-cf12-4c0c-8ac8-cab96e39cbd9

mbagalwa committed:
review index page css

marclaporte commented:
For the record: We will discuss this at today's meeting
https://github.com/cypht-org/cypht/wiki/Monthly-Community-Meetings

marclaporte commented:
I propose to have a page about the Tiki integration 

Two messages:
- If you want more features, Tiki probably has them: https://doc.tiki.org/Cypht-in-Tiki-vs-Cypht-standalone
- Cypht is easily pluggable into any PHP app (we made easy for Tiki and now it's easy for others)

This could be added soon, or later :-)

mbagalwa committed:
build draft for all pages & improove license, contribute pages

mbagalwa committed:
improove contribute page

mbagalwa committed:
update documentation page draft ui/ux

mbagalwa committed:
fix image logo path

mbagalwa committed:
build docs pages flow with md files

mbagalwa committed:
make introduction doc page draft

mbagalwa committed:
build new config-generator menu style (draft)

mbagalwa committed:
find & improve config ui/ux

mbagalwa committed:
build and improve config-generator

mbagalwa committed:
add active menu, add copy & dowload .env.config

Baraka24 committed:
Merge branch 'master' into feat/home-page-redisign

mbagalwa committed:
restyle install-methods, refactor install method pages (manual, docker, yuno & tiki)

mbagalwa committed:
improve installation page

mbagalwa committed:
improve manuel doc

mbagalwa committed:
improve terminal code

mbagalwa committed:
improve terminal card

mbagalwa committed:
Implementation of responsive design for the navbar, footer, and homepage

mbagalwa committed:
Improve responsive design for features & security page with mobile first approach

mbagalwa committed:
Localize site content to English and clean up responsive styles

• Translate remaining French copy across Markdown, CSS, JS, and Twig
• Rebuild contribute/license layouts for better mobile responsiveness
• Remove redundant copy pages

mbagalwa committed:
Make modules pages responsive

mbagalwa committed:
improve some ui/ux on module page (refine header controls, popover toggle logic, and mobile filter layout etc)

mbagalwa committed:
Review documentation structure page & make page responsive using mobile-first approach

mbagalwa committed:
refactor: update config-generator page for mobile-first responsiveness

mbagalwa committed:
delete duplicate file

mbagalwa committed:
make install page responsive

mbagalwa committed:
review colors

mbagalwa committed:
improve theme

mbagalwa committed:
fix conflit

mbagalwa committed:
Merge pull request #2 from mbagalwa/feat/dark-mode

improve theme

mbagalwa committed:
review homepage typography

mbagalwa committed:
Merge pull request #3 from mbagalwa/feat/dark-mode

Review homepage typography

mbagalwa committed:
improve theme on feature pages

mbagalwa committed:
improve theme on security page

mbagalwa committed:
Merge pull request #4 from mbagalwa/feat/dark-mode

Feat/dark mode

mbagalwa committed:
dark mode integration on install & contribute page

mbagalwa committed:
Implementation of dark mode, typography review, and UI/UX improvements on the doc page

mbagalwa committed:
Implementation of dark mode, typography review, and UI/UX improvements on the config-generator page

marclaporte commented:
@Baraka24 @mbagalwa I added a task to document the screening feature: https://github.com/cypht-org/cypht/pull/1132

Should they document here or create another branch or hold off or?

mbagalwa committed:
Implementation of dark mode, typography review, and UI/UX improvements on install & module pages

mbagalwa committed:
Merge pull request #5 from mbagalwa/feat/dark-mode

Feat/dark mode

mbagalwa committed:
fix docs folder issue

mbagalwa committed:
Merge pull request #6 from mbagalwa/feat/dark-mode

Feat/dark mode update

mbagalwa committed:
removing duplicates files & fixing certain errors

mbagalwa committed:
fix conlfit on index page

mbagalwa committed:
Merge pull request #8 from mbagalwa/feat/dark-mode

Feat/dark mode

mbagalwa committed:
improve tiki-web mail page

mbagalwa committed:
Merge pull request #9 from mbagalwa/feat/dark-mode

improve tiki-web mail page

mbagalwa committed:
refactor & improve new ui/ux for feature page

mbagalwa committed:
improve features page & add demo section of key features of cypht

mbagalwa committed:
Merge pull request #10 from mbagalwa/feat/dark-mode

Improve feature page & add demo video section

mbagalwa committed:
review footer link

mbagalwa committed:
Merge pull request #11 from mbagalwa/feat/dark-mode

review footer link

mbagalwa committed:
Review and improve Security and Install pages;
- Improve terminal code text style

mbagalwa committed:
Merge pull request #12 from mbagalwa/feat/dark-mode

Review and improve Security and Install pages;

mbagalwa committed:
build old events page & impreview event page

mbagalwa committed:
Merge pull request #13 from mbagalwa/feat/dark-mode

build old events page & impreview event page

mbagalwa committed:
feat(events): Make old events page dynamic with API and improve event details

marclaporte commented:
I am presenting Cypht on 2026-01-06 so either site is upgraded before (with enough testing) or a few days after.

https://www.rencontres-linux.quebec/en_CA/event/rencontre-linux-6-janvier-2026-31/page/introduction-rencontre-linux-decembre-2025-copie
https://www.linkedin.com/posts/rencontres-linux-au-qu%C3%A9bec_rss-cypht-imap-activity-7407119408608804865-mLF7

mbagalwa committed:
Merge pull request #14 from mbagalwa/feat/dark-mode

feat(events): Make old events page dynamic with API and improve event…

mbagalwa committed:
upadate video thumbnail url

mbagalwa committed:
upadate video thumbnail url

mbagalwa committed:
Merge pull request #15 from mbagalwa/feat/dark-mode

upadate video thumbnail url

mbagalwa committed:
Merge pull request #16 from mbagalwa/feat/dark-mode

upadate video thumbnail url

mbagalwa committed:
Make events, old-event pages responsive and improve ui/ux

mbagalwa committed:
Merge pull request #17 from mbagalwa/feat/dark-mode

Make events, old-event pages responsive and improve ui/ux

mbagalwa committed:
update events description data, change video feature to gif on key features section

mbagalwa committed:
Merge pull request #18 from mbagalwa/feat/dark-mode

update events description data, change video feature to gif on key feature section

mbagalwa committed:
revie and improve responsive on features page & some ui/ux issues

mbagalwa committed:
Merge pull request #19 from mbagalwa/feat/dark-mode

fix: improve Features page responsiveness, add missing documentation page, and resolve UI/UX issues

mbagalwa committed:
add bootstap js link remove

mbagalwa committed:
Merge pull request #20 from mbagalwa/feat/dark-mode

Added the Bootstrap JS link that had been removed

Jomagene commented:
Hello @marclaporte @Baraka24. After going through the deployed frontend (https://demo.cypht.lab15.evoludata.com/), I have provided detailed feedback UI/UX and content wise. You can find it here: https://avan.tech/item45468#threadId101097

mbagalwa committed:
Apply CSS fixes after review

mbagalwa committed:
review and update text on pages

mbagalwa committed:
update modules card link

mbagalwa committed:
Implement documentation layout, refine content structure, and enhance Introduction and Release pages

mbagalwa committed:
Add comprehensive documentation drafts for core pages

mbagalwa committed:
upadte & fix thirst review on doc page (Overview and Installation & Setup menu)

mbagalwa committed:
Review text across all pages and improve UI/UX for all documentation pages

mbagalwa committed:
Update documentation text

marclaporte commented:
I will do a final QA when you are all finished but I don't want to overlap any work, so please let me know when you are all done.

mercihabam committed:
document the screening feature

mbagalwa committed:
reviewed the site and applied final checks before push

mbagalwa committed:
fixing errors raised

mbagalwa committed:
Update security hero image, Manual install doc and fix CSS styling issues

Jomagene commented:
After a keen review you can find the second  feedback at https://avan.tech/tiki-view_tracker_item.php?itemId=45468&threadId=103181&cookietab=1#threadId103119
We've set a meeting with @mbagalwa for further details.

mbagalwa committed:
fixing errors and sugestions raised in the review #threadId103119

mbagalwa committed:
fix:review update #threadId103838

marclaporte commented:
@mbagalwa Please add a link to https://www.linkedin.com/groups/13804559

mbagalwa committed:
add linkedin link & review events list

camilevahviraki committed:
Add documentation for sieve filters

camilevahviraki committed:
Improve sieve filters documentation

mbagalwa committed:
add account setup images

mbagalwa committed:
fix: address UI/UX review feedback (config, features, install, docs, events, license)

mbagalwa committed:
fix reviews issues

mbagalwa committed:
add event cover image

mbagalwa committed:
fix swit theme mode

mbagalwa committed:
Improve hero page uniformity

mbagalwa committed:
Improve hero page uniformity

mbagalwa committed:
update event cover image

mbagalwa committed:
fix: update event cover

mbagalwa committed:
[FIX] Resolve reported issues

mbagalwa committed:
Clean up project and fix reported issues
